install.sh

```diff
@@ -11,18 +11,21 @@ name=languageclient
 
 try_curl() {
     command -v curl > /dev/null && \
-        curl --fail --location "$1" --output bin/$name
+        curl --fail --location "$1" --output bin/$name && \
+        curl --fail --location "$1.sha256" --output bin/$name.sha256
 }
 
 try_wget() {
     command -v wget > /dev/null && \
-        wget --output-document=bin/$name "$1"
+        wget --output-document=bin/$name "$1" && \
+        wget --output-document=bin/$name.sha256 "$1"
 }
 
 download() {
     echo "Downloading bin/${name}..."
     url=https://github.com/autozimu/LanguageClient-neovim/releases/download/$version/${1}
     if (try_curl "$url" || try_wget "$url"); then
+        sha256sum -c "bin/$name.sha256" || return 1
```

Comment:
Sorry for the late update.

Can you please split this PR into two? One includes generating the SHA files and the other downloading and checking SHA. Since this is modifying the installation, erroneous script will resulting release unable to install.

Another issue, can you print an error message here before `return 1`?

Thanks again for your contribution.

Reply:
No worries, yep you're right i've squashed and apply the error message on 7e8faa6eeca6f7eb0c98c64b1185c9f72f592d1f will create another PR for the generation, thanks for the feedback

Reply:
👍